thorlcr/graph/thgraphslave.cpp

```diff
@@ -2046,9 +2047,15 @@ class CLazyFileIO : public CInterfaceOf<IFileIO>
         return iFileIO.getClear();
     }
 public:
-    CLazyFileIO(CFileCache &_cache, const char *_filename, const char *_id, IActivityReplicatedFile *_repFile, bool _compressed, IExpander *_expander, const StatisticsMapping & _statMapping=diskLocalStatistics)
-        : cache(_cache), filename(_filename), id(_id), repFile(_repFile), compressed(_compressed), expander(_expander), fileStats(_statMapping)
+    CLazyFileIO(CFileCache &_cache, const char *_filename, const char *_id, IActivityReplicatedFile *_repFile, bool _compressed, IExpander *_expander, const StatisticsMapping & _statMapping, size32_t _blockedFileIOSize)
```

Comment:
This looks like it will potentially be used for non-index files too. If so, some names may be inappropriate. But not sure you really want to use for non-index files except possibly in FETCH activities, as you are probably already using large buffers anyway.

Reply:
hadn't considered fetch case, but only use case (in hthor and thor) is where it is for index read, where it is expected to potentially be read serially.
name here (in CLazyFileIO) ctor and factory are "blockedFileIOSize", so not specifically for indexes, so I think naming is ok(?), albeit it may never be used for anything but indexes.

Reply:
only used/passed through for [unfiltered] index reads at the moment.